.sync.yml

```diff
@@ -0,0 +1,15 @@
+---
+.travis.yml:
+  includes:
+  - rvm: '2.1.6'
+    sudo: required
+    services: docker
+    env: PUPPET_VERSION="3.8.4" BEAKER_set="centos-6-docker-x64"
+    script: bundle exec rake acceptance
+    bundler_args: --without development
+  - rvm: '2.1.6'
+    sudo: required
+    services: docker
+    env: PUPPET_VERSION="3.8.4" BEAKER_set="ubuntu-server-1204-docker-x64"
```

Comment:
I mean 12.04 is in LTS until 2017. We could probably do both? Maybe CentOS 6 and 7, Ubuntu 12.04 and 14.04, covers most bases.